Angular: add info about standalone components (#7321)
DevExpress/devextreme-documentation: concepts/40 Angular Components/10 Getting Started/03 Add DevExtreme to an Angular CLI Application/01 One-Command Setup.md

    npx -p devextreme-cli devextreme add devextreme-angular

> After you run the command, you can skip the following articles and move on straight to [importing DevExtreme components](/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/#Import_DevExtreme_Components).

Reviewer: ```suggestion
After you run the command, you can skip the following articles and continue with [importing DevExtreme components](/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/#Import_DevExtreme_Components).
```
